commit d912e7de52af37c25a1eb2e09fbb1fe1c100b9ed
Author: YOSHIOKA Takuma <lo48576@hard-wi.red>
Date:   2019-08-23 18:06:11 +0900

    Fix type error on getting source path
    
    It should use `.parent` instead of `.parents`.
    
    >   File "setup.py", line 39, in <module>
    >     SOURCE = (Path(__file__).parents / "pyvoicechanger.py").read_text()
    > TypeError: unsupported operand type(s) for /: '_PathParents' and 'str'

diff --git a/setup.py b/setup.py
index 7914abbd3575..0d151d200d8b 100644
--- a/setup.py
+++ b/setup.py
@@ -36,7 +36,7 @@ from pathlib import Path
 from setuptools import setup
 
 
-SOURCE = (Path(__file__).parents / "pyvoicechanger.py").read_text()
+SOURCE = (Path(__file__).parent / "pyvoicechanger.py").read_text()
 
 
 ##############################################################################
